src/pitch.ts

```diff
@@ -591,7 +591,48 @@ export class Pitch extends prebase.ProtoM21Object {
     getLowerEnharmonic(inPlace=false) {
         return this._getEnharmonicHelper(inPlace, -1);
     }
-    /* TODO: isEnharmonic, getEnharmonic, getAllCommonEnharmonics */
+
+    getEnharmonic(inPlace=false): Pitch|undefined {
+        let post = this;
+        if (!inPlace) {
+            post = this.clone();
+        }
+
+        if (post.accidental !== undefined) {
+            if (post.accidental.alter > 0) {
+                // we have a sharp, need to get the equivalent flat
+                post.getHigherEnharmonic(true);
```

Comment:
you've uncovered a bug in `_getEnharmonicHelper` which wasn't using the `inPlace` argument at all -- can you find the bug there and patch it.

Reply:
Fixed.